src/ColorPicker.jsx

```diff
@@ -0,0 +1,19 @@
+import { useRef, useEffect } from "react";
+import iro from "@jaames/iro";
```

Comment:
Yeah, if we are importing iro here like this then I am 99% sure that we can skip the html import